### DISCUSSION TITLE
Howto add your own javascript function /code using jquery and js/custom.js

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#585] [Q&A]
Dear bs-Team; 

Again maybe a 'silly' question. I back your pardon.

Your documentation says, that we can add our own js-function [that shall use/be based on the jQuery version bundled with WordPress] to the file js/custom.js of our bootscore child theme (https://bootscore.me/documentation/bootscore-child/#JavaScript)  provided, that we insert our code into the wrapper jQuery(function ($) { });

Unfortunately, that doesn't work in my installation. Do you have some very simple hello-word by which I can verify the way in general?

Many thanks for your help ... your Karsten Reincke

**Comment by crftwrk** [comment]
Sure, add `$('body').addClass('bg-success');` inside the wrapper and your body is green.

```js
jQuery(function ($) {

  // Do stuff here
  $('body').addClass('bg-success');

}); // jQuery End
```
Make sure that you have cleared the cache.

Solved?

**Reply by kreincke** [reply]
Many Thanks! Works well!! The reason for my failure? I didn't consider that I have to clear the cache ;-)

**Reply by crftwrk** [reply]
Yes, child theme has no timestamp for modified js file. That's why you must clear the cache.

**Reply by crftwrk** [reply]
Please mark the answer as solved. Thank you 👍
